libs/utils/energy.py

```diff
@@ -18,10 +18,13 @@
 import devlib
 import json
 import logging
-import time
 import os
+import psutil
+import time
```

Comment:
Is `time` needed? Below you import `sleep` from that module, probably that's already enough